store/subscription.go

```diff
@@ -259,15 +271,15 @@ func (s *subscription) insert(ctx context.Context, events []*model.Transfer) (er
 			}
 
 			// Add fee if it's a ETH event
-			if f, ok := fees[gethCommon.BytesToHash(e.TxHash)]; ok && bytes.Equal(e.Address, model.ETHBytes) {
+			if f, ok := fees[gethCommon.BytesToHash(e.TxHash)]; bytes.Equal(e.Address, model.ETHBytes) && ok {
 				if feeDiff[from] == nil {
 					feeDiff[from] = new(big.Int).Set(f)
 				} else {
 					feeDiff[from] = new(big.Int).Add(feeDiff[from], f)
 				}
 			}
 		}
-		if hasTo {
+		if hasTo && !bytes.Equal(e.From, RewardToMiner.Bytes()) && !bytes.Equal(e.From, RewardToUncle.Bytes()) {
```

Comment:
we should only record miner/uncle rewards if hasTo is true.

Reply:
I preferred the following codes.
```
if hasTo {
    if bytes.Equal(e.From, RewardToMiner.Bytes()) {
        // miner reward
    } else if bytes.Equal(e.From, RewardToUncle.Bytes() {
        // uncle reward
    } else {
        // normal transfer
    }
}
```